deploy/docs/Non_Helm_Installation.md

```diff
@@ -35,12 +35,15 @@ kubectl config use-context DESIRED_CONTEXT_NAME
 
 *Note the following steps are one way to install Helm, but in order to ensure property security, please be sure to review the [Helm documentation.](https://helm.sh/docs/using_helm/#securing-your-helm-installation)*
 
-Download Helm to generate the yaml files necessary to deploy by running
+Download latest Helm 2 to generate the yaml files necessary to deploy by running
```

Comment:
I'd like to get input from a PM on the naming convention we use. "latest Helm 2" seems odd to me, but I'm not an expert with this technology.

Reply:
`latest Helm 2 version` does that sound better?